### ISSUE TITLE
Request: Stat tracking on demand

### ISSUE THREAD

**Issue by csaloio** [Epix37/Hearthstone-Deck-Tracker#2249]
It would be great if you could have stat tracking off by default but during or possibly after a match mark that you want that game to be recorded in the stats.

**Comment by Epix37** [comment]
What would be the reason one would want to do this? It would probably not make for very good "statistics", so.. mainly for replays?

**Comment by csaloio** [comment]
Yes. There are players I'm interested in remembering (perhaps they tend to play an unusual tech card) and others I don't care about recording info about or analyzing.

**Comment by Epix37** [comment]
Without spoiling too much, there should soon be a feature that should fit your needs pretty well. Stay tuned :wink:.

**Comment by csaloio** [comment]
OK, that's great!

**Comment by csaloio** [comment]
Along similar lines it would also be cool if the UI could in some way notify you if your current opponent is someone who you've played against previously (by checking existing stats for a record containing that opponent)

**Comment by Epix37** [comment]
Since we only have the name, not the full battletag, I think that might not be possible in a reliable fashion.

**Comment by riQQ** [comment]
Couldn't AccountHi and AccountLo be used for this, if they were saved?

**Comment by csaloio** [comment]
@Epix37 Understood.

**Comment by Epix37** [comment]
@riQQ yeah, maybe. however not a feature I see happening anytime soon.
